### DISCUSSION TITLE
cant install - ComfyUI-3D-Pack

### DISCUSSION THREAD

**Discussion by iwannalikelinux** [comfyanonymous/ComfyUI#6090] [General]
Hi, still new to Linux mint and I cant install ComfyUI-3D-Pack via the manger

I run through the requirements:
pip install -r /path/to/requirements.txt

in ComfyUI-3D-Pack folder 
no issues with installing them

on starting ComfyUI > terminal states

18:56:56.727 [Warning] [ComfyUI-0/STDERR] Traceback (most recent call last):
18:56:56.727 [Warning] [ComfyUI-0/STDERR]   File "/home/office/Desktop/SwarmUI v0.9.4.0/StableSwarmUI/dlbackend/ComfyUI/nodes.py", line 2072, in load_custom_node
18:56:56.727 [Warning] [ComfyUI-0/STDERR]     module_spec.loader.exec_module(module)
18:56:56.727 [Warning] [ComfyUI-0/STDERR]   File "<frozen importlib._bootstrap_external>", line 995, in exec_module
18:56:56.728 [Warning] [ComfyUI-0/STDERR]   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
18:56:56.728 [Warning] [ComfyUI-0/STDERR]   File "/home/office/Desktop/SwarmUI v0.9.4.0/StableSwarmUI/dlbackend/ComfyUI/custom_nodes/ComfyUI-3D-Pack/__init__.py", line 53, in <module>
18:56:56.728 [Warning] [ComfyUI-0/STDERR]     module = importlib.import_module(f".{nodes_filename}", package=__name__)
18:56:56.728 [Warning] [ComfyUI-0/STDERR]              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
18:56:56.728 [Warning] [ComfyUI-0/STDERR]   File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
18:56:56.728 [Warning] [ComfyUI-0/STDERR]     return _bootstrap._gcd_import(name[level:], package, level)
18:56:56.728 [Warning] [ComfyUI-0/STDERR]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
18:56:56.729 [Warning] [ComfyUI-0/STDERR]   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
18:56:56.729 [Warning] [ComfyUI-0/STDERR]   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
18:56:56.729 [Warning] [ComfyUI-0/STDERR]   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
18:56:56.729 [Warning] [ComfyUI-0/STDERR]   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
18:56:56.729 [Warning] [ComfyUI-0/STDERR]   File "<frozen importlib._bootstrap_external>", line 995, in exec_module
18:56:56.729 [Warning] [ComfyUI-0/STDERR]   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
18:56:56.730 [Warning] [ComfyUI-0/STDERR]   File "/home/office/Desktop/SwarmUI v0.9.4.0/StableSwarmUI/dlbackend/ComfyUI/custom_nodes/ComfyUI-3D-Pack/nodes.py", line 38, in <module>
18:56:56.730 [Warning] [ComfyUI-0/STDERR]     from .mesh_processer.mesh import Mesh
18:56:56.730 [Warning] [ComfyUI-0/STDERR]   File "/home/office/Desktop/SwarmUI v0.9.4.0/StableSwarmUI/dlbackend/ComfyUI/custom_nodes/ComfyUI-3D-Pack/mesh_processer/mesh.py", line 8, in <module>
18:56:56.730 [Warning] [ComfyUI-0/STDERR]     from kiui.op import safe_normalize, dot
18:56:56.730 [Warning] [ComfyUI-0/STDERR] ModuleNotFoundError: No module named 'kiui'
18:56:56.730 [Warning] [ComfyUI-0/STDERR] 

Please can anyone help?
i want try all the3d workflows but i cant get anything to run with the above issue..

**Comment by LukeG89** [comment]
You should ask for help in the custom node repo: [ComfyUI-3D-Pack](https://github.com/MrForExample/ComfyUI-3D-Pack)

I see there is a portable version, maybe it's easier to install: https://github.com/MrForExample/ComfyUI-3D-Pack/issues/397
